### ISSUE TITLE
orange text

### ISSUE THREAD

**Issue by McKirni** [cgeo/cgeo#12822]
### Tell us your idea!

There are some system-texts that are written in orange.
You can read that very well - if you use the dark theme.
But with the light theme ... and a little bit of sun reflection ... no chance!

### What solution would you suggest?

Display this system-texts in e.g. dark blue when the light theme is activated.

**Comment by Lineflyer** [comment]
Do you have a particular screen in mind or can provide an example (screenshot)?

We had the discussion in the past: 
An option would be to darken the accent color for the white theme to improve the contrast to the background.

**Comment by McKirni** [comment]
As I said - for me personally, darkblue text on a white background would be a good contrast.
A simple change to a better readable color would be easiest to implement.
Of course, it would be more comfortable if this text color could be set individually - just as in the map-settings some colors can already be individualised.

**Comment by Lineflyer** [comment]
The theming is aligned with the c:geo logo and associated color schemes.
Therefore I would prefer to stay with a darker orange.

I will make post some examples to compare as time permits.

**Comment by Lineflyer** [comment]
While looking for a better color I tried to use the color of the shadowed orange in our logo (the part where the shadow of the c:geo box dims the orange). This would be the difference:

| current accent color | new accent color |
| --- | --- |
|![image](https://user-images.githubusercontent.com/949669/156462211-e0c9093b-7630-424d-8df6-2ea9e3c5df1f.png)|![image](https://user-images.githubusercontent.com/949669/156462143-0493234f-1044-4c6a-8c21-fed65afe5dc6.png)|

Any opinions?

**Comment by eddiemuc** [comment]
The dark orange is a better accent color on light background Imho.

That said, I too think it would be a nice idea to let the user choose his own accent color among a range of selections. I don't think it is necessary to have the accent color aligned with the logo color. The c:geo GUI style is unique enough to be recognizable even without this.

**Comment by McKirni** [comment]
I agree with eddiemuc.

**Comment by eddiemuc** [comment]
Update: also my Android system allows me to change Akzent color these days (don't know if this is specific to Oxygen OS or Android 11+ though). Maybe the time is right for something like this

**Comment by moving-bits** [comment]
> Any opinions?

I agree with that the proposed color is way better readable when using the white theme. But for me it does not really resemble a "c:geo look", it's too dark for that, IMHO. I would need to play a bit with the colors to propose an alternative (we have done that when switching to Material design, IIRC), maybe halfway between those two colors would do.

**Comment by ztNFny** [comment]
While I personally find the orange ok in terms of contrast this certainly is a usability issue, especially for visually imparied users. That said I don't like Lineflyers proposal, that brown / dirt just doesn't look like c:geo. No matter what we change it to, it'll be different than the color we use in the logo which will look "off" - unless we also change the logo, but then we have to change so many things ...

I couldn't find another issue on this since the accent color was changed, so maybe this is something that should just not be changed (based on only 1 comment)?

**Comment by ztNFny** [comment]
> The logo is not even visible in almost all c:geo screens, so why shouldn't a user be allowed to customize the color if he wants?

I'm ok with making it user-configurable - I however vote against changing the default to something that doesn't match c:geos theme. E.g. your last screenshot I'd think of the official GC app but not c:geo and that shouldn't be our default.

**Comment by eddiemuc** [comment]
> > The logo is not even visible in almost all c:geo screens, so why shouldn't a user be allowed to customize the color if he wants?
> 
> I'm ok with making it user-configurable - I however vote against changing the default to something that doesn't match c:geos theme. E.g. your last screenshot I'd think of the official GC app but not c:geo and that shouldn't be our default.

That I would fully support. Additionally, I would not allow to choose any color but only one from a curated color selection. (Problem is of course: who could create such a color selection? You definitely don't want me to do it :-)). So if similarity with other caching apps is an issue, then the too-similar colors could be left out

**Comment by ztNFny** [comment]
>  Additionally, I would not allow to choose any color but only one from a curated color selection.

Imho if we make it configurable then it should be freely configurable, don't see any reason to limit it.

**Comment by moving-bits** [comment]
IMHO we should use our default color selector in this case to stay consistent. It offers a range of 216 colors and (optionally) allows setting of opaqueness, which should be sufficient for this use-case.

**Comment by fm-sys** [comment]
FYI: Android 12 introduces the so called "Material You Design" 

If implemented my the app, it offers a new type of design that can be toggled on or off in the system settings for each app based off the user's preference. The new design would use the user's wallpaper or a configured color to determine the color scheme of the app.

Not sure how much effort it would mean for switching from "material" to "material you" though...

**Comment by Lineflyer** [comment]
Closing this stale issue from 2022.
Design evaluations and tweaks are ongoing and also the colors scheme changed multiple time since then.
Feel free to open a fresh issue if needed.
